### PR TITLE
Lk loom schema updates

### PR DESCRIPTION
**Pull request by ekiernan**
Updated Loom Schema with HCA Matrix Processing section.

### CODE REVIEW

File: website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 1]
```diff
@@ -98,3 +98,32 @@ The global attributes in the Loom apply to the whole file, not any specific part
...
 |`number_cells_detected_multiple`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells which observe more than one read of this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_detected_multiple)|
 |`number_cells_expressing`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells that detect this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_expressing)|
+
+## HCA Data Coordination Platform Matrix Processing
+Optimus was designed for processing data from the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)).
```
Comment (kbergin) [inline website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md:103]:
```suggestion
Optimus supports data processing for the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)). 
```

It was originally developed for HCA but I don't want to imply that's all it's good for.

File: website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 2]
```diff
@@ -98,3 +98,34 @@ The global attributes in the Loom apply to the whole file, not any specific part
...
+| `library_preparation_protocol.library_construction_method` | technology used for library preparation, i.e 10x or SS2 |
+| `specimen_from_organism.organ` | organ |	
+| `project.project_core.project_name` | project name |
+| `project.provenance.document_id` | project id |
+| `input_id` | metadata values for  `sequencing_process.provenance.document_id`; unique ID to demarcate the library prep |
+| `input_name` | metadata values for `sequencing_input.biomaterial_core.biomaterial_id`; unique ID for the biomaterial |
+
+The `input_id` is a comma-separated string containing the IDs for the library prep from which a cell came and can be mapped back to the unique cell barcodes that are listed in the Loom `cell_names` column attribute. 
+
+Each barcode in the `cell_names` attribute has a numerical suffix (i.e. "-1", "-2", etc.); this suffix is an index for the input IDs. For example, barcodes with "-1" comprise the first ID listed in the `input_id` string, whereas barcodes with a "-2" comprise the second ID listed in the `input_id` string.
```
Comment (jessicaway) [inline website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md:127]: these are currently 0 indexed

**Review by jessicaway — APPROVED** [review]
Minor comment about indexing the input_id and name lists (the first value corresponds to -0 suffix rather than -1 suffix)

File: website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 3]
```diff
@@ -98,3 +98,34 @@ The global attributes in the Loom apply to the whole file, not any specific part
...
+HCA DCP matrices contain HCA metadata in the Loom global attributes (see table below) which may be useful when exploring the data and linking it back to the Project metadata. Read more about each metadata field in the HCA DCP [Metadata Dictionary](https://data.humancellatlas.org/metadata). 
+
+| Metadata Attribute Name in Count Matrix | Metadata Description | 
+| --- | --- |
+| `donor_organism.genus_species` | species information; human or mouse |
+| `library_preparation_protocol.library_construction_method` | technology used for library preparation, i.e 10x or SS2 |
+| `specimen_from_organism.organ` | organ |	
+| `project.project_core.project_name` | project name |
+| `project.provenance.document_id` | project id |
+| `input_id` | metadata values for  `sequencing_process.provenance.document_id`; unique ID to demarcate the library prep |
```
Comment (khajoue2) [inline website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md:122]: We also have input_id_metadata_field and input_name_metadata_field

**Review by khajoue2 — CHANGES_REQUESTED** [review]

File: website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md (hunk 3 above)
Comment (ekiernan) [reply website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md:122]: @khajoue2 Are those being carried over into the HCA Project matrices?

**Review by khajoue2 — APPROVED** [review]

**Pull request merged**